imageroot/actions/create-module/50start-bouncer

```diff
@@ -17,18 +17,17 @@ After=${MODULE_ID}.service
 
 [Service]
 Restart=always
-ExecStartPre=/usr/bin/bash -c "if ! ipset list crowdsec-blacklists > /dev/null; then firewall-cmd --permanent --new-ipset=crowdsec-blacklists --type=hash:ip --option="timeout=0" --option="maxelem=150000"; fi"
-ExecStartPre=/usr/bin/bash -c "if ! ipset list crowdsec6-blacklists > /dev/null; then firewall-cmd --permanent --new-ipset=crowdsec6-blacklists --option=family=inet6 --type=hash:ip --option="timeout=0" --option="maxelem=150000"; fi"
-ExecStartPre=firewall-cmd --reload
-ExecStartPost=firewall-cmd --direct --add-rule ipv4 filter INPUT 0 -p all -m set --match-set crowdsec-blacklists src -j DROP
-ExecStartPost=firewall-cmd --direct --add-rule ipv6 filter INPUT 0 -p all -m set --match-set crowdsec6-blacklists src -j DROP
-ExecStopPost=firewall-cmd --direct --remove-rule ipv4 filter INPUT 0  -p all -m set --match-set crowdsec-blacklists src -j DROP
-ExecStopPost=firewall-cmd --direct --remove-rule ipv6 filter INPUT 0  -p all -m set --match-set crowdsec6-blacklists src -j DROP
+ExecStartPre=/bin/bash -c "/usr/local/sbin/firewall-rules create-ipset"
```

Comment:
Why not calling it directly like:
```suggestion
ExecStartPre=/usr/local/sbin/firewall-rules create-ipset
```

Reply:
new attempt but this time it works....probably my VM

Reply:
done